packages/framework-integration-tests/integration/helper/sleep.ts

```diff
@@ -6,7 +6,7 @@ export async function waitForIt<TResult>(
   tryFunction: () => Promise<TResult>,
   checkResult: (result: TResult) => boolean | string,
   trialDelayMs = 1000,
-  timeoutMs = 60000
+  timeoutMs = 600000
```

Comment:
I incremented all the timeouts because Azure was wonky, maybe I can turn it back, but it doesnt hurt for now